kubectl-minio/README.md

```diff
@@ -47,15 +46,32 @@ Options:
 - `--console-secret=secret-name`
 - `--cert-secret=secret-name`
 - `--image-pull-secret=`
-- `--output`
+
+You can also generate a yaml file using the `tenant generate` command.
+
+Command: `kubectl minio tenant generate --name TENANT_NAME --secret SECRET_NAME --servers SERVERS --volumes TOTAL_VOLUMES --capacity TOTAL_RAW_CAPACITY [options] > tenant.yaml`
+
+Creates a MinIO Tenant yaml based on the passed values.
+
+example: `kubectl minio tenant create --name tenant1 --secret cred-secret --servers 4 --volumes 16 --capacity 16Ti > tenant.yaml`
+
+Options:
+
+- `--namespace=minio`
+- `--image=minio/minio:RELEASE.2020-09-21T22-31-59Z`
+- `--storageClass=local`
+- `--kms-secret=secret-name`
```

Comment:
Do we have somewhere to link for how to configure the kms-secret and console-secret? If not, telling users how to specify the option doesn't provide enough guidance.

On a similar note, since KES requires initial configuration to work, I think we might need to decide how best to handle the quickstart status. From discussions with Andreas in Slack, it seems like our best way forward is to do in-memory single-replica KES for local development (with warnings about what happens if the KES pod goes down).

For production, we must advise to have a compatible KMS ready to go at start.